kazoo/tests/test_client.py

```diff
@@ -775,6 +775,19 @@ def test_create_exists(self):
         path = client.create("/1")
         self.assertRaises(NodeExistsError, client.create, path)
 
+    def test_create_stat(self):
+        if TRAVIS_ZK_VERSION:
+            version = TRAVIS_ZK_VERSION
```

Comment:
Why is this needed rather than just sticking with the `version = self.client.server_version()` for everything?

Also, a lot of projects have been moving away from Travis over the past two years, I would not be surprised if we move away at some point ourselves.

Reply:
I did not look into this one. I am not sure of the value of `TRAVIS_ZK_VERSION` vs `client.server_version()`, I simply duplicated what was present in other tests.

Let me dig a little bit...